content/en/examples/admin/signed-images/verify-all-signed-control-plane-images.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	"gopkg.in/yaml.v3"
+)
+
+func main() {
+
+	data, err := ioutil.ReadFile("list-of-all-signed-images.yaml")
+
+	if err != nil {
+
+		log.Fatal(err)
+	}
+
+	images := make(map[string][]string)
+
+	err2 := yaml.Unmarshal(data, &images)
+
+	if err2 != nil {
+
+		log.Fatal(err2)
+	}
+	domains := images["domains"]
+	names := images["names"]
+	architectures := images["architectures"]
+	versions := images["versions"]
+
+	//anything better than 4 for loops is welcome!
+	for _, d := range domains {
+		for _, n := range names {
+			for _, a := range architectures {
+				for _, v := range versions {
+					image := d + "/" + n + "-" + a + ":" + v
+					fmt.Println(image)
+				}
+			}
+		}
+	}
+}
```

Comment:
This doesn't seem like the right home for some Golang code.

Reply:
Thanks @sftim I was definitely hesitant adding it here but thought that we can always discuss it as part of the PR. @saschagrunert any thoughts on possible locations that could host this? 

Maybe https://github.com/kubernetes-sigs/promo-tools or https://github.com/kubernetes-sigs/release-sdk?

Reply:
Hm, we already verify that images are signed during the whole release process, which is something we should mention that in the document here.

I can imagine that the code lives in k/release and we make it a dedicated `krel` subcommand. Thoughts on that @puerco?

Reply:
I think we can build it into krel. The data to do the verification is already built into the SBOM so we don't have to compile the image list.

Not, that said, if we are going to put out instructions to use krel to verify, we should do two things:

1) Build the subcommand
2) Roll out a plan to release `krel` binaries so that people can fetch them to follow the instructions

If we are OK on this approach I am happy to open the issues to track it (and work on it! )

Reply:
@puerco Yay, glad to hear we do not have to maintain the yaml and image list generator. 

But, adding `krel` as a pre-requisite seems like an overhead for users to verify the images. Could we do something like this:
- `krel` subcommand for every version, generates a list of signed images during release process
- That list gets pushed into a GCS bucket or git repo like `k/release` during release of future k8s version where there is separate list for each release
- We `curl` the list from GCS bucket or git repo and provide it to `cosign` in a for loop to verify (similar to current script in this PR)

Reply:
Updated to use SBoM as source of truth for signed images